app/Models/Post.php

```diff
@@ -55,6 +56,15 @@ public function likes()
         return $this->hasMany('App\Models\Like', 'post_id');
     }
 
+    /**
+     * Lay bai post duoc like thoe user
+     */
+    public function liked($post_id) {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Models/Post.php</br>**Line**: 62</br>Opening brace should be on a new line